ansible/roles/k8s_resources/templates/monitoring-values.yml.j2

```diff
@@ -88,25 +88,25 @@ prometheus-operator:
         nginx.ingress.kubernetes.io/auth-signin: https://auth.{{ cluster_name }}/oauth2/start?rd=https://$host$escaped_request_uri
         nginx.ingress.kubernetes.io/auth-url: http://oauth2-proxy.kube-system.svc.cluster.local:4180/oauth2/auth
 
-  grafana:
-    adminUser: "{{ grafana.admin.username }}"
-    adminPassword:  "{{ grafana.admin.password }}"
-    ingress:
-      path: /
-      hosts:
-        - "grafana.{{ base_domain }}"
-        - "grafana.local.{{ base_domain }}"
-      tls:
-        - secretName: "{{ base_domain }}-tls"
-          hosts: ["grafana.{{ base_domain }}"]
-      annotations:
-        nginx.ingress.kubernetes.io/configuration-snippet: |-
-            set_escape_uri $escaped_request_uri $request_uri;
-        nginx.ingress.kubernetes.io/auth-signin: https://auth.{{ cluster_name }}/oauth2/start?rd=https://$host$escaped_request_uri
-        nginx.ingress.kubernetes.io/auth-url: http://oauth2-proxy.kube-system.svc.cluster.local:4180/oauth2/auth
-    env:
-      GF_AUTH_ANONYMOUS_ENABLED: "true"
-    setup:
-      image:
-        repository: "{{ docker_repo }}/k8s-ansible"
-        tag: "{{ legion_infra_version }}"
+legion-grafana:
+  adminUser: "{{ grafana.admin.username }}"
+  adminPassword:  "{{ grafana.admin.password }}"
+  ingress:
+    path: /
+    hosts:
+      - "grafana.{{ base_domain }}"
+      - "grafana.local.{{ base_domain }}"
```

Comment:
@mcdoker18 do we need `.local.` namespaces nowadays? I guess it was required times ago.

Reply:
Removed